packages/gatsby-plugin-mdx/src/__tests__/remark-mdx-html-plugin.ts

```diff
@@ -0,0 +1,39 @@
+import { unified } from "unified"
+import remarkMdx from "remark-mdx"
+import remarkStringify from "remark-stringify"
+import remarkParse from "remark-parse"
+
+import { remarkMdxHtmlPlugin } from "../remark-mdx-html-plugin"
+
+const source = `# Some MDX
+
+<img src="mocked"/>
+`
+
+describe(`remark: infer ToC meta`, () => {
+  it(`parses ToC and attaches it to our meta object`, async () => {
+    const processor = unified()
+      .use(remarkParse)
+      .use(remarkMdx)
+      .use(remarkMdxHtmlPlugin)
+      .use(remarkStringify)
+    const res = await processor.process(source)
```

Comment:
This runs forever and then after 5 seconds:

`Cannot handle unknown node `mdxJsxFlowElement``